exec/java-exec/src/main/java/org/apache/drill/exec/service/ServiceEngine.java

```diff
@@ -82,10 +89,32 @@ public Controller getController() {
     return controller;
   }
 
+  private void submit(ExecutorService p, final String name, final Closeable c) {
```

Comment:
Hmm... I would worry about the issues that led Guava to remove closeQuietly. As soon as possible, it would be good to switch to using try-with-resources instead of closeQuietly idioms.